File: program/find-volume-of-a-cuboid/find-volume-of-a-cuboid.kt
import kotlin.math.PI
import java.util.Scanner

fun main(args: Array<String>) {
  val scanner = Scanner(System.`in`)
  val length = scanner.nextDouble()
  val width = scanner.nextDouble()
  val height = scanner.nextDouble()

  val volume = length * width * height
  println(volume)
}

File: program/find-volume-of-a-cuboid/FindVolumeOfACuboid.cpp
#include <iostream>

using namespace std;

int main()
{
    int l, w, h, volume;
    cout << "Enter value of length : ";
    cin >> l;
    cout << "Enter value of width  : ";
    cin >> w;
    cout << "Enter value of height : ";
    cin >> h;
    volume = l * w * h;

    cout << "\nVolume of cuboid is : " << volume << endl;
    return 0;
}
